src/System.IO.IsolatedStorage/src/System.IO.IsolatedStorage.csproj

```diff
@@ -34,7 +34,7 @@
     <Reference Include="System.Diagnostics.Debug" />
     <Reference Include="System.Diagnostics.Tools" />
     <Reference Include="System.IO.FileSystem" />
-    <Reference Include="System.IO.FileSystem.AccessControl" />
+    <Reference Include="System.IO.FileSystem.AccessControl" Condition="'$(TargetGroup)' != 'uap'" />
```

Comment:
no, we have different implementation for uap, so this is actually not used any longer in uap